calculator.c

```diff
@@ -0,0 +1,114 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <string.h>
+
+#define SIZE 100
+
+int evaluation(char operator, int op1, int op2){
+    switch(operator){
+        case '+':
+            return op1 + op2; 
+        case '-': 
+            return op1 - op2;
+        case '*': 
+            return op1 * op2;
+        case '/':
+            if(op2== 0){ 
+                printf("Error: Division by zero.");
+                exit(EXIT_FAILURE);
+            }
+            else{
+                return op1 / op2; 
+            }
+        default:
+            printf("Error: Invalid operator.");
+            exit(EXIT_FAILURE);
+    }
+}
+
+
+int precedence(char operator){
+    if(operator == '+' || operator == '-'){
+        return 1;
+    }
+    if(operator == '*' || operator == '/'){
+        return 2;
+    } 
+    return 0;
+}
+
+
+int EvaluateExpression(char *expr){
+    int numbers[SIZE];
+    char operators[SIZE];
+    int nTop = -1, oTop = -1;
+    int answer;
+
+    for(int i = 0; i < strlen(expr); i++){

```

Comment:
Instead of using built-in functions like strlen(), isdigit() and isspace(), Can you create your own definition of these functions and remove unnecessary headerfile?

Reply:
Yes. I will make the changes and raise a new pull request.